### PR TITLE
Added blogpost week 0

### PR DESCRIPTION
**Pull request by tvcastillod**
Hello, here is my first blogpost.

### CODE REVIEW

File: docs/source/posts/2023/2023-06-02-week-0-tvcastillod.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+What did I do this week?
+------------------------
+
+During the community bonding period, I had the opportunity to get to know better GSoC mentors and former contributors. I also had the first meeting with FURY mentors where I got to learn more about the dynamic of the program, important things to keep in mind while coding, and suggestions when making the PRs. We also receive some details on how to work with shaders which is something I have to take into account since I will use them to develop part of the project. In addition, I’ve been working on the first part of the project which consists of the creation of tensor ellipsoids using the raymarching technique and SDFs.
+
+What is coming up next?
+-----------------------
+
+Since the advances I managed to make are in their most basic stage, I will be working this week on making changes and adjusting details for the implementation to follow fury guidelines. In this way, I plan to submit my first PR for the project to start getting feedback and making improvements. I will also start working on the first tests for the ellipsoid actor I’m working on.
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-06-02-week-0-tvcastillod.rst:22]: `fury` -> `FURY`

**Review by ganimtron-10 — COMMENTED** [review]
Hey @tvcastillod,
Please take a look at the small comment below otherwise it looks great!
Also the blog generated well!
image

I am not sure whether you have tried creating the doc locally or not, but it would be great to test generating it locally and mentioning that it generated without any error so we have an idea!

**Comment by tvcastillod** [comment]
> I am not sure whether you have tried creating the doc locally or not, but it would be great to test generating it locally and mentioning that it generated without any error so we have an idea!

 @ganimtron-10 You're right, I'll keep it in mind from now on.

**Comment by skoudoro** [comment]
Thanks @ganimtron-10 and thanks @tvcastillod for the review.

waiting for @JoaoDell review before further action

**Comment by JoaoDell** [comment]
Hey @tvcastillod you have already fixed the comment issue, so everything seems fine, good blogpost!

**Review by skoudoro — APPROVED** [review]
Thanks @tvcastillod, merging

**Pull request merged**